paasta_tools/utils.py

```diff
@@ -118,6 +118,9 @@
 
 DEFAULT_SOA_CONFIGS_GIT_URL = "sysgit.yelpcorp.com"
 
+DEFAULT_AUTOSCALING_SETPOINT = 0.8
+DEFAULT_AUTOSCALING_MOVING_AVERAGE_WINDOW = 1800
```

Comment:
might make the most sense to have these in long_running_service_tools.py since that's where the superclass of MarathonServiceConfig / KubernetesJobConfig lives. I know Maksym is trying to make utils.py less of a kitchen-sink.

Reply:
sounds good!